charts/envars-webhook/templates/mutatingwebhook.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
   labels:
     {{- include "envars-webhook.labels" . | nindent 4 }}
   annotations:
-    "helm.sh/hook": "pre-install"
+    "helm.sh/hook": "post-install,post-upgrade,pre-delete"
```

Comment:
Could you please explain the reasoning behind changing the order of resource creation. I vaguely remember the webhook had to be created before anything else.

Reply:
I changed it, because the Webhook is using the `Service` which is using the `Deployment` and there is no point in having Webhook before the Pod is actually running. It will just cause any Webhook calls by k8s control-plane to fail, because there is nothing that could respond. I can't think of a reason why would a mutatingwebhook needed to exist first.